.deploy/backup-retention-policy/Dockerfile

```diff
@@ -7,12 +7,27 @@ RUN apk add --no-cache \
 
 COPY .deploy/backup-retention-policy/script.sh /script.sh
 COPY .deploy/backup-retention-policy/entrypoint.sh /entrypoint.sh
+COPY .deploy/backup-retention-policy/healthcheck.sh /healthcheck.sh
 COPY .backup/keep.py /keep.py
 COPY scripts/cronjob /cronjob
+COPY scripts/cronjob-healthcheck /cronjob-healthcheck
 
 ENV \
   NOW="" \
-  BACKUP_RETENTION_POLICY_SCHEDULE=""
+  BACKUP_RETENTION_POLICY_SCHEDULE="" \
+  BACKUP_RETENTION_POLICY_INTERVAL="" \
+  BACKUP_RETENTION_POLICY_CMD="bash /script.sh /keep.py /backups"
+
+HEALTHCHECK \
+  --interval=21s \
+  --timeout=3s \
+  --start-period=15s \
+  --start-interval=5s \
+  --retries=2 \
```

Comment:
Backup retention policy only runs once a day or once a week. Maybe reduce frequency?

Reply:
Used `1h` as a sane default. Can be overridden with `docker`'s `--health-interval` or `compose`'s `services.*.healthcheck.interval`.